### PR TITLE
HCP Lambda + EKS Tutorial Infrastructure

### PR DESCRIPTION
**Pull request by webdog**
Ready for review

### CODE REVIEW

**Review by im2nguyen — COMMENTED** [review]
can you move everything in the `terraform` directory into the root `datacenter-deploy-hcp-eks-lambda`? it doesn't make sense for us to add another layer of directories when it's not required

for now, we're adding the eks-client module in this project as a stop-gap. when we integrate the changes into the main `eks-client` module. i would like us to move the sub-module and pull it from terraform registry

a nit -- the number of yaml files in `eks-client/service-defaults` is low-key ridiculous. we should put them into one file, delimited by `---`. this will make it easier to maintain lol

**Comment by im2nguyen** [comment]
also, please include the payments-lambda file source

**Comment by webdog** [comment]
>a nit -- the number of yaml files in eks-client/service-defaults is low-key ridiculous. we should put them into one file, delimited by ---. this will make it easier to maintain lol

Low-key I agree. Unfortunately, it's a side effect of the kubectl provider. It doesn't support multiple yaml statements delimited by dashes. I tried; didn't work.

**Comment by webdog** [comment]
* Fixed a region issue found by @eddie-rowe 
* Added lambda payments source code
* moved everything from ../terraform up one level.

cc @im2nguyen

File: datacenter-deploy-hcp-eks-lambda/lambda-payments/main.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	if err != nil {
+		return Response{StatusCode: 400, Body: "Error decoding body"}, nil
+	}
+	fmt.Println("BODY: %+v\n", requestBody)
+
+	responseBody := &ResponseBody{
+		Message:        "Payment processed successfully, card details returned for demo purposes, not for production",
+		ID:             uuid.New().String(),
+		CardPlaintext:  requestBody.Number,
+		CardCiphertext: "Encryption Enabled",
```
Comment (im2nguyen) [inline datacenter-deploy-hcp-eks-lambda/lambda-payments/main.go:57]: Please re-add Lambda to this

**Review by im2nguyen — APPROVED** [review]

File: datacenter-deploy-hcp-eks-lambda/lambda-payments/main.go (hunk 1 above)
Comment (webdog) [inline datacenter-deploy-hcp-eks-lambda/lambda-payments/main.go:57]:
```suggestion
CardCiphertext: "Encryption Enabled (Lambda)",
```

Comment (webdog) [reply datacenter-deploy-hcp-eks-lambda/lambda-payments/main.go:57]: @im2nguyen ^^

**Pull request merged**